cluster/gce/gci/configure-helper.sh

```diff
@@ -1010,6 +1010,21 @@ function start-kube-apiserver {
 
 
   local authorization_mode="RBAC"
+
+  # Create the ABAC file only if it's explicitly requested.
+  if [[ -n "${ENABLE_LEGACY_ABAC}" ]]; then
```

Comment:
Original author unavailable so this PR is updated and continued on https://github.com/kubernetes/kubernetes/pull/42197